src/gen9_vdenc.c

```diff
@@ -3456,14 +3475,16 @@ gen9_vdenc_mfx_vdenc_avc_slices(VADriverContextP ctx,
                     insert_mi_flush = 0;
                 }
 
+                pipeline_flush_params.mfx_pipeline_done = 1;
```

Comment:
Please fix the assignment to pipeline_flush_params.mfx_pipeline_done instead of a forced assignment